README.md

```diff
@@ -1,12 +1,32 @@
 LLTFI
 =====
+LLTFI (Low Level Tensor Fault Injector) is a unified SWiFI (Software-implemented fault injection) tool that supports fault injection of both C/C++ programs and ML applications written using high-level frameworks such as TensorFlow and PyTorch.
 
-LLTFI, which stands for Low Level Tensor Fault Injector, is an upgraded extension of [LLFI](https://github.com/DependableSystemsLab/LLFI), which operates on ML (TensorFlow and PyTorch) programs in addition to C/C++ programs. Please refer to the following [paper](https://blogs.ubc.ca/dependablesystemslab/2021/08/31/wip-lltfi-low-level-tensor-fault-injector/) for more information about LLTFI.
+As machine learning (ML) has become more prevalent across many critical domains, so has the need to understand ML system resilience. While there are many ML fault injectors at the application level, there has been little work enabling fault injection of ML applications at a lower level. **LLTFI** is a tool that allows users to run fault injection experiments on C/C++, TensorFlow and PyTorch applications at a lower level (at the LLVM IR level). Please refer to the following [paper](https://blogs.ubc.ca/dependablesystemslab/2021/08/31/wip-lltfi-low-level-tensor-fault-injector/) for more information about LLTFI.
 
-Because LLTFI is designed to be backwards compatible with LLFI, the basic setup instructions for LLTFI are the same as those of LLFI.
-**For instructions on how to setup and run LLTFI on ML programs, please read [this page](sample_programs/mnist).**
+LLTFI is built on top of [LLFI](https://github.com/DependableSystemsLab/LLFI) and is fully backwards compatible with LLFI. 
```

Comment:
LLTFI is built on top of [LLFI](https://github.com/DependableSystemsLab/LLFI) and is fully backwards compatible with it.